### PR TITLE
Added new shapes to simplemarker

### PR DESCRIPTION
**Pull request by ZayneTomlins**
## Added New Shapes

Added new shapes to simple markers, including: Diamond Star (4-point star), Heart, Decagon, Parallelogram (left and right versions), Trapezoid and Shield.

Added unit tests for all of the corresponding shapes.

### CODE REVIEW

**Comment by roya0045** [comment]
For the heart symbol, would it be possible to make the line go less deep in the symbol, or maybe not have an internal line. Otherwise this is a nice addition. imo.

File: resources/data/contributors.json [hunk 1]
```diff
@@ -1134,7 +1134,7 @@
       "properties": {
         "Name": "Zayne Tomlins",
         "Committer": "No",
-        "First Commit Message": "Added new ellipse marker shapes",
+        "First Commit Message": "Added more symbols to the symbolmap",
```
Comment (nyalldawson) [inline resources/data/contributors.json:1137]: Can you revert this change please?

File: src/core/symbology/qgsmarkersymbollayer.cpp [hunk 2]
```diff
@@ -735,19 +855,34 @@ bool QgsSimpleMarkerSymbolLayerBase::prepareMarkerPath( Qgis::MarkerShape symbol
       mPath.lineTo( -1, 1 );
       return true;
 
+    case Qgis::MarkerShape::Heart:
+      mPath.moveTo( 0, 0.75 );
+      //mPath.lineTo( 0.84, -0.13 );
```
Comment (nyalldawson) [inline src/core/symbology/qgsmarkersymbollayer.cpp:860]:
```suggestion
```

File: src/core/symbology/qgsmarkersymbollayer.cpp [hunk 3]
```diff
@@ -735,19 +855,34 @@ bool QgsSimpleMarkerSymbolLayerBase::prepareMarkerPath( Qgis::MarkerShape symbol
...
+      mPath.arcTo( 0, -1, 1, 1, -45, 210 );
+      mPath.arcTo( -1, -1, 1, 1, 15, 210 );
+      //mPath.lineTo( -0.84, -0.13 );
```
Comment (nyalldawson) [inline src/core/symbology/qgsmarkersymbollayer.cpp:863]:
```suggestion
```

**Comment by ZayneTomlins** [comment]
> For the heart symbol, would it be possible to make the line go less deep in the symbol, or maybe not have an internal line. Otherwise this is a nice addition. imo.


simplemarker_heart_result
Is this what you meant?

**Comment by roya0045** [comment]
> > For the heart symbol, would it be possible to make the line go less deep in the symbol, or maybe not have an internal line. Otherwise this is a nice addition. imo.
> 
> simplemarker_heart_result Is this what you meant?

Yes, I think it looks better and is more common. Thanks

**Review by nyalldawson — APPROVED** [review]
